### ISSUE TITLE
Keybase proof gets borked if you Ctrl-C the proof

### ISSUE THREAD

**Issue by strugee** [lobsters/lobsters#796]
My Keybase proof has somehow been borked. Here's the steps I remember taking (can't guarantee these are STR but it's the closest I can give):

1. Go to https://lobste.rs/settings
2. Start the Keybase proof (IIRC the page just tells you to run `keybase prove lobste.rs` but I can't really remember)
3. Enter your Lobsters username (in my case `strugee`), hit enter, and get the message: "Checking will continue while you go authorize the proof on Lobsters. To cancel hit C-c."
4. Hit Ctrl-C.
5. Run `keybase prove lobste.rs` again.
6. Answer yes at the prompt about overwriting the old proof.
7. Go through the proof flow normally.

Result: Lobsters thinks the proof went just fine, and links me to https://keybase.io/strugee/sigchain#cf80d7ce0543373eebe632ff9e2b27853394d6160ea9fa1e009ee5dfece046010f in my profile. But Keybase seems to disagree; both the proof attempts show as revoked.

At the time this happened I was on Keybase client `5.0.0`. I have since upgraded.

**Comment by pushcx** [comment]
Ping @devsnek @xgess @malgorithms who worked on this implementation - any ideas what went wrong here?

**Comment by xgess** [comment]
@strugee, @pushcx i can poke into it, but just to clarify, it looks to me like the proof is valid on both sides now. a third attempt seems to have fixed it: [keybase](https://keybase.io/strugee/sigchain#53bb7145d17a10117670f38007e634d07d0762f721773dd9517ceb92316b47810f) <--> [lobste.rs](https://lobste.rs/u/strugee).

**Comment by strugee** [comment]
Yeah sorry, should've clarified that! IIRC I deleted both ends of the proof and tried again since obviously I wanted a working proof.

**Comment by xgess** [comment]
so, neither of the lobste.rs proofs in your sigchain on keybase with the lines through them are `revoked`. it's a subtle distinction, but they're actually just `superseded`. at step 5 (posting the second proof), that's for sure when the first one got `superseded`. And the second one got `superseded` when you posted the third one. What's unclear is what state it was in from 20 dec to 24 dec.

short answer is that i can't duplicate this failure with the real services or locally. i have a couple of theories, but nothing that's a slam dunk. 

do you remember what the proof looked like in keybase during this chunk of time, and if you saw any errors? Like, was it definitely not in your client app? Was it crossed off on the website? When you were initially redirected to Keybase after accepting it in lobsters, did you see an error like `Oy! That identity proof is pending, revoked, or non-existent`? 

and thank you for the report! sorry '__'
